apps/nuxt-ssr/pages/_schema/ssr-catalogue/cohorts/index.vue

```diff
@@ -2,6 +2,7 @@
   <div class="container mt-3">
     <h1>Cohorts</h1>
     <div class="pb-1">
+      <input-search id="search" v-model="search"></input-search>
```

Comment:
I really prefer to use InputSearch instead of input-search.

Reply:
i prefer input-search